sigstore/verify/models.py

```diff
@@ -179,9 +179,9 @@ class VerificationMaterials:
     Represents the materials needed to perform a Sigstore verification.
     """
 
-    input_digest: bytes
+    hashed_input: sigstore_hashes.Hashed
```

Comment:
This seems like a breaking change :(

Reply:
Yep, it is. That's fine IMO since we're on a breakage path for 3.0 anyways, and #804 will also break compatibility here.